services/brig/test/integration/API/User/Account.hs

```diff
@@ -1454,7 +1454,10 @@ testDeleteAnonUser brig = do
 
 testDeleteWithProfilePic :: Brig -> CargoHold -> Http ()
 testDeleteWithProfilePic brig cargohold = do
-  uid <- userId <$> createAnonUser "anon" brig
+  -- A random local part, with nice, email-friendly characters
+  localPart <- cs . show <$> liftIO UUID.nextRandom
+  -- Users need to be verified if they want to upload assets, so email it is!
+  uid <- userId <$> createUserWithEmail "anon" (Email localPart "example.test") brig
```

Comment:
We should use the `randomEmail` utilitiy here.

Reply:
Will do!